### ISSUE TITLE
[DOCS] 8.5.3 Release notes

### ISSUE THREAD

**Issue by nastasha-solomon** [elastic/security-docs#2746]
### Description
Generate the 8.5.3 release notes and publish them before or on the release day, which is **December 8, 2022**.

**Milestone dates:**
- **FF:** November 29, 2022
- **BC:** November 30, 2022
- **Release:** December 8, 2022

### Steps to set up the release notes file
1. Check out the `main` branch.
2. Run a `git fetch origin` then `git pull` to pull in the latest changes.
3. Create a branch for the release notes, for example, `issue-XXX-8.5.3-rn`.
4. Add a link to the `8.5.3` release notes to update the release note's index page. Should be: 
`<<release-notes-8.5.3, {elastic-sec} version 8.5.3>>`
5. Make necessary changes and then open a PR.

### Notes for release notes PR
* Remember to add the `8.5` label so the release notes are backported properly.

**Comment by nastasha-solomon** [comment]
Add the following Endpoint bug fix to the release notes:

> Fixed Endpoint crash on Linux that could occur on busy systems when network events or malicious behavior protection are enabled

The PR is private, so won't be linking to it.

**Comment by tinnytintin10** [comment]
Pre 8.5.3, if users installed the KSPM integration, deployed elastic-agent to their k8s cluster, and were successfully generating findings **but** didn't have the correction permissions to view certain indices in ES, they would see unhelpful screens like `403 Unauthorized` or even a `Findings delayed` empty state. With the release of 8.5.3, we've remedied this with https://github.com/elastic/kibana/pull/145794 where now users will see an empty state that makes them aware of their lack of permissions. We've also updated the [getting started section of the KSPM docs](https://www.elastic.co/guide/en/security/current/get-started-with-kspm.html) to inform users of these permissions requirements too https://github.com/elastic/security-docs/pull/2741

@nastasha-solomon can we update the release notes to also include this bug fix? A potential text for it could what I have captured below, feel free to adjust it as you see fit. 

> Fixed loading state for both the posture dashboard and findings page to inform users if they lacked the necessary permissions to view posture data. https://github.com/elastic/kibana/pull/145794

I will be out this coming week so if you have any questions between now and Tuesday, feel free to ping the #cloud-security-posture channel or Tehila directly.
